app/renderer/components/bookmarks/bookmarksToolbar.js

```diff
@@ -86,8 +86,9 @@ class BookmarksToolbar extends ImmutableComponent {
       }
     }
     if (e.dataTransfer.files.length > 0) {
```

Comment:
so we didn't replace files object with items, but we only added items into dataTransfer?